src/ReportGenerator.Core/Reporting/Builders/Rendering/HtmlRenderer.cs

```diff
@@ -299,7 +299,7 @@ public void TestMethods(IEnumerable<TestMethod> testMethods, IEnumerable<FileAna
                             codeElement.FirstLine,
                             codeElement.CoverageQuota.HasValue ? coverageRounded.ToString() : "undefined",
                             codeElement.CoverageQuota.HasValue ? ReportResources.Coverage2 + " " + codeElement.CoverageQuota.Value.ToString(CultureInfo.InvariantCulture) + "% - " : string.Empty,
-                            WebUtility.HtmlEncode(codeElement.Name),
+                            WebUtility.HtmlEncode($"File {item.Key} => " + codeElement.Name),
```

Comment:
Assign an index to each file on the class coverage page. So that later we can refer to this index for each method instance.

Reply:
I like the idea, but in the sidebar the horizontal space is limited. I would prefer to show the file index only in the tooltip.